src/components/Player.vue

```diff
@@ -640,14 +664,50 @@ li.move:not(.from) .player .overlay svg.move {
   border-radius: 10px;
   top: 5px;
   box-shadow: 0 0 5px black;
-  text-overflow: ellipsis;
-  overflow: hidden;
   padding: 0 4px;
 
+  span {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+
   #townsquare:not(.spectator) &:hover,
   &.active {
     color: red;
   }
+
+  &:hover .pronouns {
+    opacity: 1;
+    color: white;
+  }
+
+  .pronouns {
+    display: inline-flex;
+    position: absolute;
+    right: 120%;
+    max-width: 250px;
+    z-index: 25;
+    background: rgba(0, 0, 0, 0.5);
+    border-radius: 10px;
+    border: 3px solid black;
+    filter: drop-shadow(0 4px 6px rgba(0, 0, 0, 0.5));
+    align-items: center;
+    pointer-events: none;
+    opacity: 0;
+    transition: opacity 200ms ease-in-out;
```

Comment:
```suggestion
    transition: opacity 200ms ease-in-out;
    padding: 0 4px;
    bottom: -3px;
```
That brings it closer in look and position to the name.